tests/test_covariance_fourier_ssc_fsky.py

```diff
@@ -260,6 +260,8 @@ def test_get_covariance_block_WL_benchmark(cov_fssc):
 
     # Modify tjpcov instance
     cov_fssc.io.sacc_file = s
+    # Create an empty mask dict since there are no input mask files in the yaml
+    cov_fssc.mask_files = {}
```

Comment:
Why do yo need this for the fsky approx?

Reply:
Ok. Instead of passing a disc mask, you should call your SSCFsky class and pass fsky=0.05, as the docstring says.